docker-compose.yml

```diff
@@ -264,15 +264,15 @@ services:
       context: ./cron
       args:
         BASE_IMAGE: "$SNUBA_IMAGE"
-    command: '"*/5 * * * * gosu snuba snuba cleanup --storage errors --dry-run False"'
+    command: '"*/5 * * * * /usr/sbin/gosu snuba snuba cleanup --storage errors --dry-run False"'
```

Comment:
This is not related to this PR and we're not still sure if there is a problem at all, as no one else has reported this issue until now. Let's keep the discussion in https://github.com/getsentry/self-hosted/issues/1556.

Reply:
My mistake, yep. Removed this commit from PR.